lucet-runtime/lucet-runtime-internals/src/instance.rs

```diff
@@ -533,10 +533,15 @@ impl Instance {
     /// modified by this call; it is the embedder's responsibility to clear or reset their state if
     /// necessary.
     ///
+    /// This will also reinitialize the kill state, which means that any outstanding
+    /// [`KillSwitch`](struct.KillSwitch.html) objects will be unable to terminate this instance.
+    /// It is the embedder's responsibility to initialize new killswitches after resetting an
```

Comment:
```suggestion
    /// It is the embedder's responsibility to initialize new `KillSwitch`es after resetting an
```
just to be clear to the reader that we do mean this library item, `KillSwitch`